lib/livebook_web/live/session_live.ex

```diff
@@ -1701,6 +1703,21 @@ defmodule LivebookWeb.SessionLive do
      push_patch(socket, to: ~p"/sessions/#{socket.assigns.session.id}/settings/custom-view")}
   end
 
+  def handle_event(
+        "set_smart_cell_editor_intellisense_node",
+        %{"js_view_ref" => cell_id, "node" => node, "cookie" => cookie},
+        socket
+      ) do
+    Session.set_cell_attributes(socket.assigns.session.pid, cell_id, %{
+      editor_intellisense_node:
+        if is_binary(node) and node =~ "@" and is_binary(cookie) and cookie != "" do
+          {node, cookie}
```

Comment:
@jonatanklosko, we chose to keep them as strings here so we don't perform unsafe conversion of strings to atoms in Livebook itself, only in the runtime.

Reply:
I considered that initially and didn't like the inconsistency, but it is more sensible so all good :)